src/forms/Form/Form.tsx

```diff
@@ -82,12 +74,10 @@ const Form = (props: FormProps) => {
 							title={title}
 							type={type}
 							description={description}
-							onSubmit={(e) => submit(e)}
-							submitButtonAttrs={submitButtonAttrs}
 							onCancel={onCancel ? (e) => cancel(e) : null}
-							cancelButtonAttrs={cancelButtonAttrs}
 							sections={sections}
 							view={view}
+							buttons={buttons}

```

Comment:
I feel we could filter the buttons here, that way we don't care if it's a form drawer or a regular form, or if the user resizes, the filteredButtons stay the same and it's only processed once. It could be inside a useMemo that only updates when buttons change.